docs/api/browser-window.md

```diff
@@ -169,6 +169,15 @@ Emitted when window is minimized.
 
 Emitted when window is restored from minimized state.
 
+### Event: 'resize'
+
+Emitted when window is getting resized on all platforms.
+
+### Event: 'move'
+
+Emitted once when the window is moved to a new position on `OSX` and emitted during the process 
```

Comment:
We usually don't add <code>\`</code> around operating system names, so `OS X` instead of <code>\`OSX\`</code>.